README_JAVAC_EXPERIMENT.md

```diff
@@ -0,0 +1,57 @@
+# JDT over Javac
```

Comment:
move the file to o.e.j.c.javac/README.md please

Reply:
Ack